components/devfile-page/HeaderTags.tsx

```diff
@@ -13,39 +13,42 @@ interface Props {
  */
 const HeaderTags = ({ tags }: Props) => {
   const numOfVisibleTags = 8;
-  if (!tags) {
+  if (!tags || tags.length === 0) {
```

Comment:
I think its a little easier to read `!tags?.length` and a little faster.

Reply:
!tags returns a boolean which doesn't have the length property so this doesn't work

Reply:
`!tags?.length` shoudl work because this is how its evaluated.

1. `?.` checks if the `length` property is defined
2. `length` checks returns a truthy value if the length is greater than 1
3. `!` inverses the truthy value of length

Reply:
<img width="521" alt="Screen Shot 2021-08-09 at 9 42 32 AM" src="https://user-images.githubusercontent.com/31493292/128742573-d61a69c8-41a1-4516-9db7-7f4369d1ef67.png">

Reply:
The if statement should look like this.
``` js
  if (!tags?.length) {
```

Reply:
oh i see

Reply:
resolved